Source/UI/About/IdentityView.swift

```diff
@@ -126,7 +136,10 @@ struct IdentityView: View {
                     }
                     .zIndex(extendedHeader ? 500 : 1000)
                     .offset(y: headerOffset)
-                MessageListView(strategy: strategy)
+                MessageStack(dataSource: dataSource)
```

Comment:
I find it a little confusing to have two components named `MessageList` and `MessageStack`. They both sound like Views but one is not. Maybe this set of renames would make things more clear? LMK what you think:
- MessageList -> MessageListDataSource
- InfiniteList -> InfiniteListDataSource
- InfiniteStackView -> InfiniteStack (to be consistent with MessageStack)

Reply:
👌